src/main/window-manager.ts

```diff
@@ -73,7 +74,7 @@ export class WindowManager {
       await this.showSplash();
       await this.mainView.loadURL(`http://localhost:${this.proxyPort}`)
       this.mainView.show();
-      this.splashWindow.hide();
+      this.splashWindow.close();
```

Comment:
Hiding splashWindow leaves window in background causing troubles in integration tests where windowCount is calculated.